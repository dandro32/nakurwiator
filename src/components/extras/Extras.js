import React from 'react';
import Dashboard from '../shared/dashboard/Dashboard';

const Extras = () => {
  const cards = [
    {
      title: 'Dodaj nowe białeczko',
      desc: 'Dodaj nową masóweczkę na buły',
      href: '/extras/proteins/new',
      hrefTitle: 'Dodej',
      imgSrc: '/img/proteins_new.jpg'
    },
    {
      title: 'Lista białeczek',
      desc: 'Zobacz jakie mosz białeczka, edytuj i usuwaj',
      href: '/extras/proteins/list',
      hrefTitle: 'Lista białeczek',
      imgSrc: '/img/proteins_list.jpg'
    }
  ];
  return (<Dashboard cards={cards} />);
};

export default Extras;