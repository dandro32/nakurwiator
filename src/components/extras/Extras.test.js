import React from 'react';
import Extras from './Extras';
import TestRenderer from 'react-test-renderer';

jest.mock('../UI/dashboard/Dashboard', () => 'Dashboard component');

describe('Extras component tests', ()=> {
  let result;

  beforeEach(() => {
    result = TestRenderer.create(
      <Extras />
    ).toTree();

  });

  it('renders correctly', () => {
    expect(result).toMatchSnapshot();
  });
});