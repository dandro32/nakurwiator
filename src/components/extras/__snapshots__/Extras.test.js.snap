// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Extras component tests renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "cards": Array [
        Object {
          "desc": "Dodaj nową masóweczkę na buły",
          "href": "/extras/proteins/new",
          "hrefTitle": "Dodej",
          "imgSrc": "/img/proteins_new.jpg",
          "title": "Dodaj nowe białeczko",
        },
        Object {
          "desc": "Zobacz jakie mosz białeczka, edytuj i usuwaj",
          "href": "/extras/proteins/list",
          "hrefTitle": "Lista białeczek",
          "imgSrc": "/img/proteins_list.jpg",
          "title": "Lista białeczek",
        },
      ],
    },
    "rendered": Array [],
    "type": "Dashboard component",
  },
  "type": [Function],
}
`;
